`tests/rustdoc-gui/go-to-collapsed-elem.goml` might be flaky

```
======== tests/rustdoc-gui/go-to-collapsed-elem.goml ========
[ERROR] line 40
    at `tests/rustdoc-gui/go-to-collapsed-elem.goml` line 21: Error: Node is detached from document: for command `click: "//*[@id='search']//a[@href='../test_docs/struct.Foo.html#method.must_use']"`
    at <file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/test_docs/struct.Foo.html?search=t_use>
<= doc-ui tests done: 143 succeeded, 1 failed, 0 filtered out
Error: ()
```

https://github.com/rust-lang/rust/pull/157744#issuecomment-4677858066

Note: I was testing `windows-2025-vs2026` runners, but AFAIK this shouldn't be relevant to the VS version.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "browser-ui-test": "^0.24.0",
+    "browser-ui-test": "^0.24.1",
     "es-check": "^9.4.4",
     "eslint": "^8.57.1",
     "typescript": "^5.8.3"

